plugins/modules/aci_hsrp_interface_policy.py

```diff
@@ -0,0 +1,305 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2023, Shreyas Srish (@shrsr) <ssrish@cisco.com>
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {"metadata_version": "1.1", "status": ["preview"], "supported_by": "certified"}
+
+DOCUMENTATION = r"""
+---
+module: aci_hsrp_interface_policy
+short_description: Manage HSRP interface policies (hsrp:IfPol)
+description:
+- Manage HSRP interface policies on Cisco ACI fabrics.
+options:
+  tenant:
+    description:
+    - The name of the Tenant the hsrp interface policy should belong to.
+    type: str
+    aliases: [ tenant_name ]
+  hsrp:
+    description:
+    - The HSRP interface policy name.
+    type: str
+    aliases: [ hsrp_interface, name ]
+  description:
+    description:
+    - The description of the HSRP interface.
+    type: str
+    aliases: [ descr ]
+  controls:
+    description:
+    - The interface policy controls.
+    type: list
+    elements: str
+    choices: [ bfd, bia ]
+  delay:
+    description:
+    - The administrative port delay of HSRP interface policy.
+    type: int
+  reload_delay:
+    description:
+    - The option for reload delay of HSRP interface policy.
```

Comment:
can you add the range of the values these attribute can take?

Reply:
ok